packages/jsii-python-runtime/src/jsii/_kernel/__init__.py

```diff
@@ -101,18 +101,29 @@ def _recursize_dereference(kernel, d):
 def _dereferenced(fn):
     @functools.wraps(fn)
     def wrapped(kernel, *args, **kwargs):
-        return _recursize_dereference(kernel, fn(kernel, *args, **kwargs))
+        output = fn(kernel, *args, **kwargs)
+        print(output)
```

Comment:
probably don't want to leave this in there

Reply:
Tisk :)